### PR TITLE
Ticket 4 completed

### PR DESCRIPTION
**Pull request by charlie-storer-vensta**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -59,4 +59,22 @@ describe("Returns with the correct JSON endpoint", () => {
...
+      .then((response) => {
+        response.body.review.forEach((review) => {
+          expect(typeof review.title).toBe("string");
+          expect(typeof review.created_at).toBe("string");
+          expect(typeof review.votes).toBe("number");
+          const input = Object.keys(
+            endpoint["GET /api/reviews/:review_id"].exampleResponse
+          );
+          const expectedOutput = Object.keys(response.body);
+          expect(input).toMatchObject(expectedOutput);
```
Comment (MKD01) [inline __tests__/api.test.js]: What is this testing for?

File: __tests__/api.test.js [hunk 2]
```diff
@@ -59,4 +59,22 @@ describe("Returns with the correct JSON endpoint", () => {
...
+  test("/api/reviews/:review_id", () => {
+    const review_id = 3;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(200)
+      .then((response) => {
+        response.body.review.forEach((review) => {
+          expect(typeof review.title).toBe("string");
+          expect(typeof review.created_at).toBe("string");
+          expect(typeof review.votes).toBe("number");
```
Comment (MKD01) [inline __tests__/api.test.js]: This should include all properties. You can see a list of the expected properties in your ticket on trello

File: __tests__/api.test.js [hunk 3]
```diff
@@ -59,4 +59,22 @@ describe("Returns with the correct JSON endpoint", () => {
         expect(input).toMatchObject(expectedOutput);
       });
   });
+  test("/api/reviews/:review_id", () => {
+    const review_id = 3;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(200)
+      .then((response) => {
+        response.body.review.forEach((review) => {
```
Comment (MKD01) [inline __tests__/api.test.js]: Should review be an array?

**Review by MKD01 — CHANGES_REQUESTED** [review]
Good work so far, but there are a few things to change/add, once you have made all your changes push them up to this branch and then re-submit a review request via /nchelp pr

**Review by Beth-McCormack — APPROVED** [review]

**Pull request merged**